### ISSUE TITLE
“Add Export/Sync to VSCode for Smoother Project Development”

### ISSUE THREAD

**Issue by jirojirocirociro** [stackblitz/bolt.new#522]
Is your feature request related to a problem? Please describe:

I love how fast and easy it is to prototype with Bolt.new in the browser, but once the projects start getting more complex, things get a bit buggy and slow. It’s hard to keep using WebContainer for bigger tasks, and I often end up needing a more stable environment for debugging and advanced features.

Describe the solution you’d like:

It would be awesome to have a feature where we can export our projects to VSCode or even sync them with a local VSCode setup. This way, I could use Bolt.new’s AI for quick prototyping, and when things get more complex, I can just switch to VSCode to continue working on my project. The sync could keep everything up-to-date between the browser and VSCode, making it super smooth to move back and forth.

Describe alternatives you’ve considered:

One alternative is sticking with WebContainer, but that would mean a lot of improvements are needed to make it more stable. Another option could be building an extension for VSCode, but honestly, I think a hybrid approach—starting in the browser and moving to VSCode when needed—would give the best of both worlds.

**Comment by sarah-robbins** [comment]
I would also **love** to see this added!

**Comment by richiedhubbard88** [comment]
Yes this is certainly needed! Especially when testing API endpoints - some reason (assume security) bolt.new won't interact with any endpoints, I'm needing to download the zip to computer, then run, check issues and report back - takes so long.

**Comment by DavidMagdi** [comment]
i think i'm not gonna renew my subscription unless find way to chat on vscode and. sync between browser and my local machine

**Comment by ovasylenko** [comment]
I think it should be solved by exporting the code as a GitHub repository. Also, vote for this feature.

**Comment by kamal0808** [comment]
I was able to tie the download from bolt directly to my local development and github repo via some AI-written shell scripts. It works pretty well. 
But yes this as a feature is needed

**Comment by Christian-onsai** [comment]
+1
